tests/python/opinfos.py

```diff
@@ -1355,11 +1356,39 @@ def scaled_grouped_mm_wrapper(
         ),
     )
 
+    def scaled_mm_wrapper(mat1, mat2, scale1, scale2, alpha, bias, beta, dtype):
+        assert beta is None
+        return torch._scaled_mm(
+            mat1, mat2, scale1, scale2, bias, alpha, out_dtype=dtype
+        )
+
+    scaled_mm_opinfo = OpInfo(
+        lambda fd: fd.ops.scaled_mm,
+        "scaled_mm",
+        # limit test to mxfp8 for now
```

Comment:
Would you think it makes sense to add conventional fp8 tests? I mean, test cases of fp8 data matrices with per-tensor scales